src/hooks/useParticipants.ts

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
```

Comment:
I messed up earlier moving useProjects to src/hooks. Keeping this with the component context works just fine. Thoughts?